### DISCUSSION TITLE
hi chaithanya im getting this error while accessing template file

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1856] [Q&A]
![Screenshot (35)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/a3c0fc47-11c3-49a9-b832-f04dba0d8118)

**Comment by chaithanya1812** [comment]
In your code
syntax mistake  in hosts section 
![20240310_183935](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/85e05511-171c-448b-a442-3d69fb69de53)
